c2corg_ui/static/js/editing/documentediting.js

```diff
@@ -670,7 +671,65 @@ app.DocumentEditingController.prototype.openModal = function(selector, sizem) {
 app.DocumentEditingController.prototype.presetQuality = function(doc) {
   // Do nothing special in the standard editing controller.
   // Will be overridden in inheriting controllers.
-  return 1;
+  let score = 0;
```

Comment:
That's not very consistent: it's said above that the fonction does nothing and should be overriden in inheriting controllers but you add some code below.
That would be more consistent to keep the function as it was and to move your code to inheriting controllers (create them if not already existing).